source/includes/_ais.md

```diff
@@ -185,6 +189,10 @@ any more.
 
 We've implemented version 3.1.2 of the [Open Banking Direct Debits specification](https://openbanking.atlassian.net/wiki/spaces/DZ/pages/1077805417/Direct+Debits+v3.1.2).
 
+We have only implemented `GET /direct-debits` endpoint, and not the account-specific endpoints.
```

Comment:
Other way around 🙈 , we've implemented the account-specific endpoint but not the general one